### ISSUE TITLE
HTML from nbconvert looks completely different from HTML in presentation mode

### ISSUE THREAD

**Issue by johann-petrak** [damianavila/RISE#617]
I have used jupyter nbconvert to html slides (using `--to slides`) in the past to get a HTML file that works pretty much as the presentation started from within the notebook does.

However when I look at the result of the conversion again, there are three main differences:

* all the Python code cells have a very small font and a huge border around the code (this is really bad in the generated HTML and works fine in the presentation from the notebook).
* the size of the slide, including all fonts etc. changes in the generated HTML file, but not in the presentation from the notebook (I actually like the behavior of the HTML file better here)
* the slide/fragement number is shown in the presentation from the notebook, but no int the generated HTML file

I am using:
* RISE version 5.7.1
* IPython 7.34.0
* notebook 6.4.12
* ipykernel 6.15.0

I found it really hard to find any information about how to influence those font sizes either in the presentation from the notebook and even less in the HTML created with nbconvert - or if this is even possible. 
Is there any information on this? 

An most importantly: is it possible to get the code cells show in the result of the nbconvert conversion as they look in the presentation from the notebok?

**Comment by johann-petrak** [comment]
Screenshots, same browser window size when (1) presenting from the notebook and (2) from the HTML created with nbcovert

## (1) From notebook
![rise-from-nb](https://user-images.githubusercontent.com/619106/175658525-c030b471-2d30-471c-89c2-03a153aa5774.png)
## (2) From result of nbconvert
![rise-from-nbconvert](https://user-images.githubusercontent.com/619106/175658551-bc22e2fc-7a6a-40cd-8f41-a05460a44156.png)

**Comment by johann-petrak** [comment]
I also notice that the fonts in the generated HTML do not react to the normal way of how one can increase/decrease font size in the browser (Ctrl + in Firefox): while the scroll controls get huge, the shown text (both in the code cell and cell output) do not change or change only a tiny bit.

**Comment by johann-petrak** [comment]
Please note that what I get when running `jupyter nbconvert --to slides myslides.ipynb` looks very different from the example shown in the README of this projectt, where the code cells are not surrounded by such a huge border and not as tiny! 

Is there any idea what is going on and how to make the code cells more readable? I was hoping to use this for preparing a course, but this is really something that makes using RISE problematic.

**Comment by parmentelat** [comment]
Hi there

indeed the way the slideshow appears in the browser has little to do with the way your notebook will be converted through `nbconvert`; in particular, the latter will miss many css injections, so this is not expected to work at all

as a teacher, I do use RISE in class, but also produce handouts through jupyter-book; the presentation is completely different from the slideshow, which is fine and imo better suited to when students come back to that content, where they need a searchable content, not slides.

my $0.02

**Comment by johann-petrak** [comment]
@parmentelat not sure if you are referring to the `--to slides` option for `jupyter nbconvert` -- I sure would have expected that this is expected to work, what else would `--to slides` be intended for?

**Comment by parmentelat** [comment]
yeah, well, let me rephrase: it might have worked that way a very long while ago then ;)

**Comment by johann-petrak** [comment]
Hmm I still do not really understand the relationship between the RISE package and nbconvert slide support.

However, I think the reason for the odd styling was this: while I updated jupyter and notebook, the nbconvert package in the environment I used was not at the latest version (6.5.0) but at version 6.3.0. After upgrading the code cells look ok. 

I guess this happens because dependency management in Python with pip is just awful: one never explicitly installs nbconvert and thinks upgrading what was installed (notebook, jupyter) will bring everything up to date, when in reality, this is not what is happening: the old nbconvert package was still used.  

Did not have anything to do with the RISE package if I understand correctly, so sorry for the spam.
